android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -371,18 +371,22 @@ public void onLeScan(final BluetoothDevice device, final int rssi, final byte[]
           @Override
           public void run() {
             UriBeacon uriBeacon = UriBeacon.parseFromBytes(scanResult.getScanRecord().getBytes());
```

Comment:
Maybe I'm missing something but it seems that there is no need to create a `ScanResult`.